web-animations/interfaces/AnimationTimeline/document-timeline.html

```diff
@@ -20,6 +21,17 @@
 }, 'document.timeline identity tests');
 
 promise_test(function(t) {
+  var baseTime = document.timeline.currentTime;
+  var iframeBaseTime = iframe.contentDocument.timeline.currentTime;
```

Comment:
It might be clearer to use
var iframe = document.getElementById('iframe')
as in the above test, even if it is not strictly necessary.

Is there a way to dynamically create the iframe like other test files are doing with createDiv?